robyn/router.py

```diff
@@ -84,6 +84,7 @@ def get_routes(self) -> List[Route]:
 class MiddlewareRouter(BaseRouter):
     def __init__(self) -> None:
         super().__init__()
+        self.globals = {"BEFORE_REQUEST": [], "AFTER_REQUEST": []}
```

Comment:
one nit - can we rename this to `self.global_event_handlers`? `globals` actually means `global` variables in python stack terms